ui/i18n/src/main/res/values/strings.xml

```diff
@@ -521,13 +521,16 @@
     <string name="pref_enqueue_downloaded_summary">Add downloaded episodes to the queue</string>
     <string name="pref_skip_silence_title">Skip silence in audio</string>
     <string name="behavior">Behavior</string>
+    <string name="episode_lists">Episode lists</string>
     <string name="pref_default_page">Default page</string>
     <string name="pref_default_page_sum">Screen that is opened when starting AntennaPod</string>
     <string name="pref_back_button_opens_drawer">Back button opens drawer</string>
     <string name="pref_back_button_opens_drawer_summary">Pressing the back button on the default page opens the navigation drawer</string>
     <string name="remember_last_page">Remember last page</string>
     <string name="pref_delete_removes_from_queue_title">Delete removes from queue</string>
     <string name="pref_delete_removes_from_queue_sum">Automatically remove an episode from the queue when it is deleted</string>
+    <string name="pref_downloads_button_action_title">Play/Pause button on Downloads screen</string>
+    <string name="pref_downloads_button_action_sum">Display a Play/Pause button instead of a Delete button</string>
```

Comment:
I think we can leave out the "pause". This makes it a bit shorter. I think it is quite obvious that the play button becomes pause when playing, without having to explicitly state this on the settings screen